contrib/systemd/gitea.service

```
@@ -20,6 +20,9 @@ Type=simple
 User=git
 Group=git
 WorkingDirectory=/var/lib/gitea/
+# If using unix socket: Tells Systemd to create /run/gitea folder to home gitea.sock
+# Manual cration would vanish after reboot.
+#RuntimeDirectory=gitea
```

I might be wrong, but isn't it more like...?:
```suggestion
RuntimeDirectory=gitea
```

---

I've left it commented out for optional usage only if you choose to use sockets.